src/business.c

```diff
@@ -247,6 +247,8 @@ void ins_sign_ssh_blob(uint8_t p1, uint8_t p2, uint8_t *dataBuffer, uint32_t dat
 {
     bool getPublicKey = ((p2 & P2_PUBLIC_KEY_MARKER) != 0);
     p2 &= ~P2_PUBLIC_KEY_MARKER;
+    bool last = ((p1 & P1_LAST_MARKER) != 0);
```

Comment:
This is a change in behavior bringing non retrocompatibles changes? I mean this means updating the app force updating the agent? Is this really needed? Should we update a doc or changelog somewhere?

Reply:
discussed in slack. needed change